release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,381 @@
+
+//ROX-26546
+[id="view-the-violation-status-directly-on-the-violations-page_{context}"]
+=== View the violation status directly on the Violations page
+
+With {product-title-short} 4.7, you can now see the status of a violation directly on the *Violations* page so that you can quickly determine whether the violation is still active. This streamlines automation workflows, such as creating a Jira ticket and sending it to an owner who does not use {product-title-short} regularly. 
+
+By following the link in the ticket, the owner can immediately see if the violation is still relevant, reducing the risk of unnecessary delays or deprioritization. In addition, the page provides the full context of the violation and ensures that all relevant details are immediately available.
+
+//ROX-26335
+[id="prioritize-cves-with-the-epss-integration_{context}"]
+=== Prioritize CVEs with the EPSS integration
+
+{product-title-short} 4.7 introduces integration with the Exploit Prediction Scoring System (EPSS), a data-driven model that estimates the likelihood of a software vulnerability being exploited. 
+
+In addition to the severity and Common Vulnerability Scoring System (CVSS) score, an EPSS probability score from 0%-100% is now displayed for detected Common Vulnerabilities and Exposures (CVEs). 
+You can use the EPSS score to better prioritize the remediation of CVE vulnerabilities and strengthen your security strategy.
+
+//tech preview
+//ROX-26330
+[id="gain-visibility-into-the-external-ips-in-the-network-graph_{context}"]
+=== Enhanced visibility into the external IPs in the network graph
+
+:FeatureName: Visualizing external entities
+include::snippets/technology-preview.adoc[]
+
+With {product-title-short} 4.7, you can now get a better insight into the external IPs behind external entities in the network graph. 
+This enhancement helps you identify the specific IP addresses that your deployments are communicating with and gives you better insight into the external interactions and potential risks.
+
+For more information, see xref:../operating/visualizing-external-entities.adoc#visualizing-external-entities[Visualizing external entities].
+
+//ROX-25007
+[id="enhanced-options-for-the-roxctl-netpol-generate-command_{context}"]
+=== Enhanced options for the roxctl netpol generate command
+
+In {product-title-short} 4.7, the `roxctl netpol generate` command automatically detects when DNS connections are required and generates them accordingly. If you do not specify a port, port `53` is selected automatically, but you can change this by using the `--dnsport` flag. You can now specify the port names not only as numbers, but also as text. For example, you can use `--dnsport dns`. 
```

Comment:
Hi Agil, not sure where you find the original text to be unclear/unfit but this text to me, is still less clear than the original. The points I'm trying to make are 
1.  This flag was already supported. We only added the named port option
2. If you are an OpenShift customer you *should* (not *can*) change the port number